modules/tenant-management/src/Volo.Abp.TenantManagement.Blazor/Pages/TenantManagement/TenantManagement.razor.cs

```diff
@@ -127,4 +126,9 @@ protected override ValueTask SetTableColumnsAsync()
 
         return base.SetTableColumnsAsync();
     }
+
+    protected virtual void ShowHidePassword(bool isShowPassword)
```

Comment:
no need parameter `isShowPassword` here.
simple:
```csharp
protected virtual void ShowHidePassword()
{
    IsShowPassword = !IsShowPassword;
}
```

Reply:
It's simpler but not a safe behaviour

Reply:
sorry, can you explain what's mean of **not a safe behaviour**

Reply:
yw, to avoid multiple instances